central/graphql/resolvers/plotted_image_vulnerabilities.go

```diff
@@ -0,0 +1,62 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+func init() {
+	schema := getBuilder()
+	utils.Must(
+		schema.AddType("PlottedImageVulnerabilities", []string{
+			"basicImageVulnerabilityCounter: VulnerabilityCounter!",
+			"imageVulnerabilities(pagination: Pagination): [ImageVulnerability]!",
+		}),
+	)
+}
+
+// PlottedImageVulnerabilitiesResolver returns the data required by top risky images scatter-plot on vuln mgmt dashboard
+type PlottedImageVulnerabilitiesResolver struct {
+	root    *Resolver
+	all     []string
+	fixable int
+}
+
+func newPlottedImageVulnerabilitiesResolver(ctx context.Context, root *Resolver, args RawQuery) (*PlottedImageVulnerabilitiesResolver, error) {
+	query := withImageCveTypeFiltering(args.String())
+	allCveIds, fixableCount, err := getPlottedVulnsIdsAndFixableCount(ctx, root, RawQuery{Query: &query})
```

Comment:
Just a note @keyallis  and @charmik-redhat, the `getPlottedVulnsIdsAndFixableCount` uses the legacy datastore, which we will need to address. In postgres, new image vulns and node vulns should be used.